app/setup.py

```diff
@@ -178,8 +178,7 @@ def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-compl
         # no cache and silence warning
         cache.init_app(application, config={'CACHE_NO_NULL_WARNING': True})
 
-    # Switch off flask default autoescaping as content is html encoded
-    # during schema/metadata/summary context (and navigition) generation
+    # Switch off flask default autoescaping as schema content can contain html
     application.jinja_env.autoescape = False
```

Comment:
I didn't realise we turned this off globally. Flask docs recommend against this approach and instead recommend turning off autoescape on a per block basis. Won't block this PR but worth noting. https://flask.palletsprojects.com/en/1.1.x/templating/#controlling-autoescaping

Reply:
Before autoescaping was turned off `safe` was used throughout the templates to turn off escaping for strings that could contain HTML. Most strings in templates used this approach. This lead to safe being used everywhere without any thought about whether the contents had been escaped or not. By making the choice explicit it made it clearer what should be escaped. 

Also, changing the default to autoescaping on now could have a performance impact - https://jinja.palletsprojects.com/en/2.10.x/faq/#why-is-autoescaping-not-the-default.